browse/static/js/coreRecommender.js

```diff
@@ -0,0 +1,62 @@
+function getCORERecCookie() {
+    var coreCookie = "arxiv_core";
+    var dc = document.cookie;
+    var prefix = coreCookie + "=";
+    var begin = dc.indexOf(prefix);
+    var end = document.cookie.indexOf(";", begin);
+    var returnVal;
+    if (end == -1) {
```

Comment:
Prefer strong typed comparison `===`, sometimes behaviour of `==` may be unexpected.